### ISSUE TITLE
Type Error with Deno v1.6.0

### ISSUE THREAD

**Issue by jazzdan** [kt3k/deno_license_checker#8]
Looks like this:

```
error: TS2794 [ERROR]: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?
setTimeout(() => resolve(), n);
                      ~~~~~~~~~
     at https://deno.land/x/license_checker@v3.1.0/util.ts:36:22

     An argument for 'value' was not provided.
        new <T>(executor: (resolve: (value: T | PromiseLike<T>) => void, reject: (reason?: any) => void) => void): Promise<T>;
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~
        at asset:///lib.es2015.promise.d.ts:33:34
```

I'm not super familiar with Deno, so I'm not sure how easy a fix it but I thought I'd leave a note here.

Thanks so much for this super useful module!

**Comment by kt3k** [comment]
@jazzdan Thank you for using this tool and thank you for reporting the issue! Fixed the issue at #9. Please try the latest verson https://deno.land/x/license_checker@v3.1.1

**Comment by jazzdan** [comment]
Thanks @kt3k!
